models.go

```diff
@@ -61,6 +61,26 @@ type CompletionRequest struct {
 	Stream bool `json:"stream,omitempty"`
 }
 
+// EditsRequest is a request for the edits API
+type EditsRequest struct {
+	// ID of the model to use. You can use the List models API to see all of your available models, or see our Model overview for descriptions of them.
+	Model string `json:"model"`
+	// The input text to use as a starting point for the edit.
+	Input string `json:"input"`
+	// The instruction that tells the model how to edit the prompt.
+	Instruction string `json:"instruction"`
+	// Sampling temperature to use
+	Temperature *float32 `json:"temperature,omitempty"`
+	// Alternative to temperature for nucleus sampling
+	TopP *float32 `json:"top_p,omitempty"`
+	// How many edits to generate for the input and instruction. Defaults to 1
+	N *int `json:"n"`
+
+	// Whether to stream back results or not. Don't set this value in the request yourself
+	// as it will be overriden depending on if you use CompletionStream or Completion methods.
+	Stream bool `json:"stream,omitempty"`
```

Comment:
does the stream parameter work with the edits API too or is this left over from copying the completion request?

Reply:
Well noted. I force pushed another version of this commit that doesn't add the Stream field